Menus/start_screen.gd

```diff
@@ -7,6 +7,19 @@ const template_version: String = "0.1"
 @onready var version_num: Label = %VersionNum
 func _ready() -> void:
 	version_num.text = "v%s" % template_version
+	# we don't have a saved window position
```

Comment:
This block also feels like it should be pushed to the specific windows settings script so the internals of that logic lives where the rest of that code is and then we've got one line here that initializes this either on the start screen or possibly even as part of the Globals ready, since the Start screen may not always be the entry point to the project (could be a cutscene or some other credits/startups screen that precedes it). Thoughts?

Reply:
moved to `Globals.display_manager.restore_window()`